operational/flexera/custom_branding/flexera_custom_branding.pt

```diff
@@ -0,0 +1,352 @@
+name "Configure Custom Branding"
+rs_pt_ver 20180301
+type "policy"
+short_description "Custom branding of an organisation for both MSP and Enterprise customers. See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/flexera/custom_branding/) and [docs.flexera.com/flexera/EN/Administration](https://docs.flexera.com/flexera/EN/Administration/PartnerMSPCustomizations.htm) to learn more."

```

Comment:
Short description, like the title, should indicate what the policy template does. "Configures custom branding"

Also, organization should be spelled with a 'z'. I realize this is entirely regional, and this isn't a comment on which dialect of English is better, but American English is the English used for Flexera and thus spelling should conform to that where possible for consistency.